com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicPointsCloud.cs

```diff
@@ -1,32 +1,52 @@
-using UnityEngine;
+using System.Collections.Generic;
 using UnityEngine.Formats.Alembic.Sdk;
 
 
 namespace UnityEngine.Formats.Alembic.Importer
 {
+    /// <summary>
+    /// This class is the scene data container for animated point clouds.
+    /// </summary>
     [ExecuteInEditMode]
-    internal class AlembicPointsCloud : MonoBehaviour
+    public class AlembicPointsCloud : MonoBehaviour
     {
         // members
-        [ReadOnly] public PinnedList<Vector3> m_points = new PinnedList<Vector3>();
-        [ReadOnly] public PinnedList<Vector3> m_velocities = new PinnedList<Vector3>();
-        [ReadOnly] public PinnedList<uint> m_ids = new PinnedList<uint>();
-
-        [ReadOnly] public Vector3 m_boundsCenter;
-        [ReadOnly] public Vector3 m_boundsExtents;
+        PinnedList<Vector3> m_points = new PinnedList<Vector3>();
+        PinnedList<Vector3> m_velocities = new PinnedList<Vector3>();
+        PinnedList<uint> m_ids = new PinnedList<uint>();
 
         internal AlembicPoints m_abc = null;
 
         [Tooltip("Sort points by distance from sortFrom object")]
-        public bool m_sort = false;
-        public Transform m_sortFrom;
+        internal bool m_sort = false;
+        internal Transform m_sortFrom;
 
         // properties
-        internal AlembicPoints abcPoints { get { return m_abc; } }
-        public PinnedList<Vector3> points { get { return m_points; } }
-        public PinnedList<Vector3> velocities { get { return m_velocities; } }
-        public PinnedList<uint> ids { get { return m_ids; } }
+        internal PinnedList<Vector3> pointsList { get { return m_points; } }
+        internal PinnedList<Vector3> velocitiesList { get { return m_velocities; } }
+        internal PinnedList<uint> idsList { get { return m_ids; } }
```

Comment:
Ok... now I'm confused. In previous class, member variable were not prefixed with `m_` but here yes, and accessor were all TitleCase but here no. Is it an inconsistency or because it's a MonoBehavior? Maybe convention are different.